inc/Print.php

```diff
@@ -599,6 +599,7 @@ public static function AdminEditUser()
 			}
 			// Get user data
 			$userData = $GLOBALS['db']->fetch('SELECT * FROM users WHERE id = ? LIMIT 1', $_GET['id']);
+			$lastScoreDetection = $GLOBALS['db']->fetch('SELECT * FROM score_detections ORDER BY created_at DESC LIMIT 1');

```

Comment:
* -> created_at?

Reply:
yeah, i was trying to not reveal the shape of this table initially but i can at least do this query specifically